### ISSUE TITLE
Patch for /trunk/CASUALcore/src/CASUAL/resources/Translations/Russian.properties

### ISSUE THREAD

**Issue by GoogleCodeExporter** [N004Drake/android-casual#6]
```
Found one line not in its place, corrected
```

Original issue reported on code.google.com by `chernog...@gmail.com` on 29 Sep 2013 at 3:53
- Merged into: #5

Attachments:
- [English.properties.patch](https://storage.googleapis.com/google-code-attachments/android-casual/issue-6/comment-0/English.properties.patch)

**Comment by GoogleCodeExporter** [comment]
```
what application are you using for patching?

```

Original comment by `adamoutler@gmail.com` on 29 Sep 2013 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
I just translated English.properties in Dreamweaver and pasted back in edit 
window - it looks the same as English text now.
```

Original comment by `chernog...@gmail.com` on 29 Sep 2013 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
Merged with issue #5.
```

Original comment by `log...@logatech.org` on 12 Oct 2013 at 12:04
- Changed state: **Duplicate**
